src/hyperfine/export/mod.rs

```diff
@@ -0,0 +1,94 @@
+mod csv;
+
+use self::csv::CsvExporter;
+
+use std::io::Result;
+
+/// The ExportEntry is the main set of values that will
+/// be exported to files when requested.
+#[derive(Debug, Default, Clone, Serialize)]
+pub struct ExportEntry {
+    /// The command that was run
+    command: String,
+    /// The mean run time
+    mean: f64,
```

Comment:
Can we use `Second` instead of `f64` here? Does this work with `Serialize`?

Reply:
I'll check